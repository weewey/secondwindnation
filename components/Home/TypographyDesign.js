import React from 'react';
import Typography from 'material-ui/Typography';
import PropTypes from 'prop-types';
import { withStyles } from 'material-ui/styles';
import Hidden from 'material-ui/Hidden';

const styleSheet = ({
  subheading: {
    marginTop: '10vh',
    letterSpacing: '-0.03em',
    fontSize: 32,
    color: '#322',
    lineHeight: 1.2,
    fontWeight: '300',
    position: 'relative',
    right: '32vw',
  },
  mobileSubHeading: {
    marginTop: 20,
    letterSpacing: '-0.03em',
    fontSize: 32,
    color: '#322',
    lineHeight: 1.2,
    fontWeight: '300',
  },
});

const TypographyDesign = props => (
  <div>
    <Hidden smDown>
      <Typography className={props.classes.subheading} type="title">
        Our Latest Articles
      </Typography>
    </Hidden>
    <Hidden mdUp>
      <Typography className={props.classes.mobileSubHeading} type="title">
        Our Latest Articles
      </Typography>
    </Hidden>
  </div>
);

TypographyDesign.propTypes = {
  classes: PropTypes.object.isRequired,
};

export default withStyles(styleSheet)(TypographyDesign);
