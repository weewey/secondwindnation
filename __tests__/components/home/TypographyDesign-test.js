import React from 'react';
import renderer from 'react-test-renderer';

import TypographyDesign from '../../../components/Home/TypographyDesign';

describe('Home Page', () => {
  it('should render', () => {
    const typo = renderer.create(
      <TypographyDesign />,
    );
    const tree = typo.toJSON();
    expect(tree).toMatchSnapshot();
  },
  );
});
