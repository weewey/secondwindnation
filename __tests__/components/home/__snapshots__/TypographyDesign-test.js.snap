// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home Page should render 1`] = `
<div>
  <h2
    className="MuiTypography-root-3 MuiTypography-title-9 TypographyDesign-subheading-1"
  >
    Our Latest Articles
  </h2>
</div>
`;
